<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recent Searches</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/searches.png</normaloff>icons/searches.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>15</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Recent Searches:-</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>831</width>
      <height>471</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>City Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Temperature</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Humidity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Pressure</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date_Time</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
